lib/plugins/sfCompat10Plugin/lib/config/sfValidatorConfigHandler.class.php

```diff
@@ -104,7 +104,7 @@ public function execute($configFiles)
 
     $this->generateRegistration('GET', $data, $methods, $names, $validators);
 
-    if (count($fillin))
+    if ($fillin !== null && count($fillin))
```

Comment:
This check is far from complete, because objects can't be counted unless they implement `\Countable` or are iterable (or array for older versions). Could be that I'm missing a case.